modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/deep/pagination/configuration/DeepPaginationConfigurationWrapper.java

```diff
@@ -1,61 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.portal.search.elasticsearch7.internal.deep.pagination.configuration;
-
-import com.liferay.petra.reflect.ReflectionUtil;
-import com.liferay.portal.configuration.module.configuration.ConfigurationProvider;
-import com.liferay.portal.kernel.module.configuration.ConfigurationException;
-import com.liferay.portal.search.elasticsearch7.configuration.DeepPaginationConfiguration;
-
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
-/**
- * @author Gustavo Lima
- */
-@Component(
-	configurationPid = "com.liferay.portal.search.elasticsearch7.configuration.DeepPaginationConfiguration",
```

Comment:
@Valmir-unicap After moving logic from this component to `ElasticsearchIndexSearcher`, this `configurationPid` thing is not moved. Is it confirmed to be redundant?